UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -416,14 +416,14 @@ private static string HighlightInGameName(string input, bool playSound = true)
 
 	private static string HighLightCodeWords(string input)
 	{
-		if (ThemeManager.ChatHighlight == false || PlayerManager.LocalPlayerScript == null) return input;
+
+		if (PlayerManager.LocalPlayerScript == null) return input;
+
 		if (PlayerManager.LocalPlayerScript.PossessingMind == null) return input;
-		if (PlayerManager.LocalPlayerScript.PossessingMind.IsAntag == false) return input;
 
-		SpawnedAntag antag = PlayerManager.LocalMindScript.GetAntag();
+		if (PlayerManager.LocalPlayerScript.PossessingMind.IsAntag == false) return input;
```

Comment:
why does it need to go through the player script first?

Reply:
Habit from before mind rework, corrected.